src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--sun calendar--month-length--31">
+      <div class="calendar__items"></div>
```

Comment:
```suggestion
      <div class="calendar__item"></div>
```
кожен div - це item, а не items. (однина, а не множина)